From ddf5485cbe25c7fa04970748a76f562e2b98ef19 Mon Sep 17 00:00:00 2001
From: Imre Kaloz <kaloz@openwrt.org>
Date: Thu, 16 Mar 2023 19:52:56 +0530
Subject: [PATCH 131/500] OpenWrt: 651-wireless_mesh_header.patch

mac80211: increase wireless mesh header size

lede-commit 3d4466cfd8f75f717efdb1f96fdde3c70d865fc1
Signed-off-by: Imre Kaloz <kaloz@openwrt.org>

Change-Id: Ib03127a80a967a3a0a18de294f2f8a72cc51ae8f
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
(cherry picked from commit 3e29cc4d361dfb29bd45fc4c8e95daa3094d0f7c)
---
 include/linux/netdevice.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index a3b25cecebde..63ce71611f99 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -157,8 +157,8 @@ static inline bool dev_xmit_complete(int rc)
 
 #if defined(CONFIG_HYPERV_NET)
 # define LL_MAX_HEADER 128
-#elif defined(CONFIG_WLAN) || IS_ENABLED(CONFIG_AX25)
-# if defined(CONFIG_MAC80211_MESH)
+#elif defined(CONFIG_WLAN) || IS_ENABLED(CONFIG_AX25) || 1
+# if defined(CONFIG_MAC80211_MESH) || 1
 #  define LL_MAX_HEADER 128
 # else
 #  define LL_MAX_HEADER 96
-- 
2.34.1

